heron/tools/cli/src/python/submit.py

```diff
@@ -112,8 +116,12 @@ def launch_a_topology(cl_args, tmp_dir, topology_file, topology_defn_file, topol
       extra_jars=extra_jars,
       args=args,
       java_defines=[])
-  err_context = "Failed to launch topology '%s'" % topology_name
-  succ_context = "Successfully launched topology '%s'" % topology_name
+  if "dry_run" in cl_args:
```

Comment:
We could remove this and just append ` in dry run mode` to the end of the existing messages if we're in dry-run mode.